### ISSUE TITLE
npm build issue 

### ISSUE THREAD

**Issue by p3777dev** [amcharts/amcharts5#1239]
![build issue](https://github.com/amcharts/amcharts5/assets/153706487/5309eef5-38e4-4dd6-9de9-663b43328cff)

I am trying to build from source. When I run `npm run build` it successfully runs build. 

However, when I use the `dist` folder to import amchart these two modules are not present in their respective directories.
When I manually move them from src, they seem to be working.

Is this expected or am I missing something?

**Comment by Pauan** [comment]
Thanks for the report, `npm run build` should be all that you need to do, we'll investigate this.

**Comment by workingbuddy10** [comment]
That would be great ! @Pauan 

I was also experiencing something similar while working on build.

**Comment by p3777dev** [comment]
Thank you 

> Thanks for the report, `npm run build` should be all that you need to do, we'll investigate this.

Thank you for responding .

**Comment by workingbuddy10** [comment]
> Thanks for the report, `npm run build` should be all that you need to do, we'll investigate this.

Any update on this?
@Pauan 

Thanks!

**Comment by Pauan** [comment]
@workingbuddy10 Sorry for the delay, it's been a pretty hectic holiday season.

I tried creating a fresh clone of [the amcharts5 repo](https://github.com/amcharts/amcharts5). Then I did `npm install`. Then I did `npm run build`.

But `amcharts5/dist/es2015/.internal/charts/venn/vennjs/index.js` does correctly exist, and `amcharts5/dist/es2015/.internal/bundled/xlsx/xlsx.js` also exists.

I also tried running our `venn-diagram` example using the `dist/es2015` folder, and it worked fine without any errors.

How are you testing the code?
